client/src/Context/Autherization.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect, createContext } from 'react';
+import Axios from 'axios';
+
+const ContextApi = createContext();
+
+function Authenticator({ children }) {
+  const [id, setId] = useState('');
+  const [isAuth, setIsAuth] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const fetchData = async () => {
+    try {
+      const { data } = await Axios('api/v1/isauth');
+      setIsAuth(data.isAuth);
+      if (isAuth) {
```

Comment:
I'm not sure this will do exactly what you expect - `setIsAuth` is asynchronous so probably you call `setIsAuth(true)` but on the next line `isAuth` will still be false. You could validate this by logging out some values. 

It's an easy change to make though, you can use `data.isAuth` instead on this line and it'll behave as expected.